feed-server/labels.ts

```diff
@@ -0,0 +1 @@
+../labels.ts
```

Comment:
I suspect this is a weird thing that was added and left here for who knows what reason?  Unless maybe it's a symlink?? that would make sense, just doublecheck